aten/src/ATen/native/MathBitsFallback.h

```diff
@@ -44,6 +44,9 @@ struct MathOpFallback {
     const auto stack_start = stack->size() - num_arguments;
```

Comment:
The comment above these lines (which gives the general algorithm) should get updated with the new algo